src/WebJobs.Script.WebHost/WebJobsApplicationBuilderExtension.cs

```diff
@@ -38,7 +38,7 @@ public static IApplicationBuilder UseWebJobsScriptHost(this IApplicationBuilder
             builder.UseMiddleware<HttpRequestBodySizeMiddleware>();
             builder.UseMiddleware<SystemTraceMiddleware>();
             builder.UseMiddleware<HostnameFixupMiddleware>();
-            if (environment.IsAnyLinuxConsumption() || environment.IsKubernetesManagedHosting())
+            if (environment.IsAnyLinuxConsumption() || environment.IsKubernetesManagedHosting() || environment.IsManagedAppEnvironment())
```

Comment:
Similar to how we introduced IsAnyLinuxConsumption, let's add an IsAnyKubernetesEnvironment check to simplify the currently 2 places where we're doing the check for Kubernetes